### ISSUE TITLE
Rename to ProtonUp?

### ISSUE THREAD

**Issue by trymeouteh** [DavidoTek/ProtonUp-Qt#88]
Rename to ProtonUp and drop the "-Qt" from the name?

**Comment by DavidoTek** [comment]
Hmm, I don't know. ProtonUp is another tool which inspired ProtonUp-Qt and ProtonDown sounds like "downgrading" Proton...
I thought about renaming ProtonUp-Qt, but the tool is already known under that name and I didn't come up with a better name.

**Comment by teppyboy** [comment]
ProtonUp is already nice since we got our little fame with that name, changing it would just cause to lost that fame and we'll have to build it again.

**Comment by Doomer-Cat** [comment]
We need ProtonLeft and ProtonRight too

**Comment by Aegeontis** [comment]
Maybe just ProtonManager? Although there already is a tool with such a name at: https://github.com/thelocomotion/GloriousProtonManager, it hasnt been updated in 2 years. Perhaps someone could get in touch with thelocomotion, so that they may archive their repo and link to this one.

**Comment by sonic2kk** [comment]
There is a good argument that ProtonUp-Qt already has name recognition though, not to mention internal naming that would need refactored.

Not to mention sites like GamingOnLinux frequently refer to ProtonUp-Qt, and downstream projects like SteamTinkerLaunch would need to be updated to use the new name.

These are not strict reasons against a rename, but more that we should be mindful of a rename and would need a strong reason to do it because of the obstacles it could create.

**Comment by Aegeontis** [comment]
While I cant speak to the codebase and how much refactoring it would need, the downstream name usage is not really an issue. When a repo is renamed on github, the old name automatically becomes a link to the new one. So if someone somewhere has a link, it will point to the correct repo, even if the link uses the old name.

There is also the option of archiving this repo and moving the code into an organization. That way the old repo will still contain the old name and therefore a google search for "ProtonUp-qt", will still point to it. Then the old repo's readme will just have a small message pointing to the new repo under the org.

As for reasons: 
1. The current name is not really self-explanatory:
* "proton" in the name makes sense
* "Up" doesnt make sense (down would make sense, but I agree with @DavidoTek that it sounds like downgrading) or at least I dont know why "Up" is in the name (and most people probably also dont)
* IMO toolkits don't look too great as part of an app name (but as I said, thats just my opinion)
2. "ProtonManager" (or similar) explains exactly what the app does:
* ProtonUp-Qt -> Some app that does something with proton
* ProtonManager -> An app that manages proton

**Comment by sonic2kk** [comment]
> So if someone somewhere has a link, it will point to the correct repo, even if the link uses the old name.

Of course. To be clearer, I was referring to actual named references, not links.

<hr>

> "Up" doesnt make sense

"Upgrade". Not sure why most people wouldn't know, I would find that strange. And given the reactions to OP, I don't agree with the sentiment that most people don't know.

> IMO toolkits don't look too great as part of an app name (but as I said, thats just my opinion)

I disagree. This has been Linux convention for a while, and it is good to note the toolkit directly in the name. This can be useful for a variety of reasons, including people that may want to only use Qt-based or only use GTK-based applications.

I think if a rename does happen, having the toolkit in the name at the very least should be non-negotiable.

> "ProtonManager" (or similar) explains exactly what the app does

Although it supports a huge amount more than just Proton, so if we want to go that road, neither the current name nor this name makes sense.

<hr>

I think the current application name is fine. I understand why some people might want a rename, but I think ProtonUp-Qt is a very clean name. I think the "Up" makes sense, because it allows you to upgrade Proton versions (although again, the project does a lot more than this now). It also keeps an imo respectful homage to the project that this was originally based off of.

I think the current name is well known, sensible, and not worth the effort to "rebrand." This feels like a solution in need of a problem.
